data/themes/default/style.css

```diff
@@ -532,6 +532,22 @@ TimeLine {
 	font-size: 8px;
 }
 
+TimeLineWidget {
+	/* font-size only supports px and pt. */
+	font-size: 7pt;
+	/* lengths also support em. This will make sure that the height
+	   will always change in the same proportion as the font size
+	   defined above.
+	   If you want a fixed size set min and max to the same value. */
+	min-height: 1.5em;
+	max-height: 1.5em;
+
```

Comment:
You want to add this here: 
```css
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
 						stop: 0 #8796a7, stop: 1.0 #3e454e );
```

to reinstate the old look. 

Also, shouldn't
```css
TimeLine {
	font-size: 8px;
}
```

be removed?